build_test.go

```diff
@@ -41,22 +43,13 @@ func testBuild(t *testing.T, context spec.G, it spec.S) {
 		Expect(err).NotTo(HaveOccurred())
 
 		buffer = bytes.NewBuffer(nil)
-		logger := scribe.NewEmitter(buffer)
+		logger := scribe.NewEmitter(buffer).WithLevel("DEBUG")
```

Comment:
Personally I don't think there is much value in unit testing the log lines. The assertions on the tests are sufficient to determine whether we are entering the correct path.

Reply:
@joshuatcasey what do you think? Did you want to keep the log assertions or remove them?

Reply:
In this case I like having the log statements tested for two reasons:

1. The output from the logs is meant for the consumer, and the debug logs are for that time when the consumer can't figure out what's going on. In that sense I feel like they form some sort of external contract. Typical app logs are meant for consumption by the producing developers, so the contract there can change.
2. We do often test logs (both non-debug and debug) in the integration tests, so it feels like we can tighten the feedback loop around changed logs with unit tests. [Composer-install example](https://github.com/paketo-buildpacks/composer-install/blob/main/integration/default_app_test.go)